python/rmm/tests/test_rmm.py

```diff
@@ -871,3 +871,22 @@ def func_with_arg(x):
     rmm.unregister_reinitialize_hook(func_with_arg)
     rmm.reinitialize()
     assert L == [2]
+
+
+@pytest.mark.parametrize(
+    "cuda_ary",
+    [
+        lambda: rmm.DeviceBuffer.to_device(b"abc"),
+        lambda: cuda.to_device(np.array([97, 98, 99, 0, 0], dtype="u1")),
+    ],
+)
+def test_rmm_device_buffer_copy(cuda_ary):
+    cuda_ary = cuda_ary()
+    db = rmm.DeviceBuffer.to_device(np.zeros(5, dtype="u1"))
+    db.copy_from_device(cuda_ary)
+    db_copy = db.copy()
```

Comment:
Like I suggested in the docstring example above, this test should make some assertions about the copy. Something like:
```python
assert db is not db_copy
assert db.ptr != db_copy.ptr
assert len(db) == len(db_copy)
```

Reply:
Done